cmd/interfaces.go

```diff
@@ -6,6 +6,7 @@ import (
 
 type runner interface {
 	SetDir(d string)
+	SetEnv(e []string)
```

Comment:
Is there a constraint on naming this method in this interface, i.e. because it then implements a certain other interface? Or could just as well call it `AppendEnv`?

Reply:
We can also name it AppendEnv as well. Since the scope of that PR is get remove the envExec runner (... and not perform some renamings under the hood) I would like to suggest to open another PR in case you think it should be urgently renamed.

Reply:
Ok.